gen.go

```diff
@@ -10,6 +10,10 @@ type Generator struct {
 	g *group
 }
 
+func (g *Generator) Group() *group {
```

Comment:
It's by design that `*group` is not leaked directly. Is there a specific reason for this?

Reply:
Got it, I encountered a case, in which I prefer to split operations into multi functions like this:

``` go
type MyStruct {
        g *gg.Generator
        other fields ......
}

func (m *MyStruct) GenerateCode() {
        m.g = gg.New()
        m.GeneratePackage()
        m.GenerateCodeGroup1()
        m.GenerateCodeGroup2()
        m.Save()
}

func (m *MyStruct) GenerateCodeGroup() {
        m.g.Group().NewFunction().
                .AddBlock()
}

func (m *MyStruct) Save() {
       m.g.WriteFile(...)
}
```

So I think the `*group` field inside `Generator` can be accessed in public. But I think it's OK to the new method. Since it's unrelated to the PR.